snorkel/labeling/model/label_model.py

```diff
@@ -881,7 +913,7 @@ def fit(
 
         self._set_constants(L_shift)
         self._set_class_balance(class_balance, Y_dev)
-        self._create_tree()
+        self._set_dependencies([])
```

Comment:
can we default `deps = [ ]` in the `_set_dependencies` function? or pass in `(`deps = [ ])`
and nit: can we call it `_set_structure()` or something that's more general than just dependencies?